python/pyspark/sql/tests.py

```diff
@@ -201,6 +202,28 @@ def assertPandasEqual(self, expected, result):
                "\n\nResult:\n%s\n%s" % (result, result.dtypes))
         self.assertTrue(expected.equals(result), msg=msg)
 
+    @contextmanager
```

Comment:
This was extracted alone from https://github.com/apache/spark/commit/d6632d185e147fcbe6724545488ad80dce20277e